qa/common/common.sh

```diff
@@ -430,6 +430,7 @@ EOF
 function ceph_apparmor {
     salt '*' state.apply ceph.apparmor
     salt '*' cmd.run 'systemctl | grep -i apparmor'
+    salt '*' cmd.run '/usr/sbin/apparmor_status'
```

Comment:
Imho it would be better to use ```/usr/sbin/aa-status``` here. ```/usr/sbin/apparmor_status``` is a symlink to ```aa-status```...might be suse specific..?

Reply:
So use `aa-status` here, too? https://github.com/SUSE/DeepSea/blob/d8a1c2d01b88a5bdf4b5069c1a941377777ed6fd/srv/modules/runners/ready.py#L74

Reply:
yeah thats probably a good idea as well.